aten/src/ATen/native/quantized/cpu/tensor_operators.cpp

```diff
@@ -59,7 +59,16 @@ AT_FORALL_OPERATORS(DEFINE_COMPARATOR)
 #undef AT_FORALL_OPERATORS
 #undef DEFINE_COMPARATOR
 
-Tensor& quantized_resize_cpu_(Tensor& self, IntArrayRef size) {
+Tensor& quantized_resize_cpu_(
+    Tensor& self,
+    IntArrayRef size,
+    c10::optional<MemoryFormat> optional_memory_format) {
+  auto memory_format =
```

Comment:
same as the named tensor case, shouldn't this check for not having a value?

Because this behavior differs from non-quantized tensor, where we will actually make something contiguous even if the size matches.